modules/ebi-metagenomics/sanntis/main.nf

```
@@ -1,34 +1,70 @@
 process SANNTIS {
-    tag "$meta.id"
+    tag "${meta.id}"
     label 'process_high'
 
     conda "${moduleDir}/environment.yml"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'biocontainers/sanntis:0.9.3.5--pyhdfd78af_1' :
-        'biocontainers/sanntis:0.9.3.5--pyhdfd78af_1' }"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'biocontainers/sanntis:0.9.4.0--pyhdfd78af_0'
+        : 'biocontainers/sanntis:0.9.4.0--pyhdfd78af_0'}"
 
     input:
-    tuple val(meta), path(interproscan_output), path(gbk)
+    tuple val(meta), path(interproscan), path(gbk), path(faa)
 
     output:
-    tuple val(meta), path("*_sanntis.gff"), emit: gff
-    path "versions.yml"                   , emit: versions
+    tuple val(meta), path("*_sanntis.gff.gz"), emit: gff
+    path "versions.yml", emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
+    if (faa && gbk) {
```

Since we are at it, maybe throw the same error if both faa and gbk were empty?